.ci/release.groovy

```diff
@@ -70,6 +71,11 @@ pipeline {
   }
 }
 
+def changeDescription(){
+  currentBuild.description = "Rollout the environment ${params.environment}"
+  currentBuild.displayName = "#${BUILD_NUMBER}-(${params.environment})"
```

Comment:
What other information do we have available? Do we have the commit message or something like this?

Reply:
we do not have the code cloned, but we can grab the Docker image we are going to deploy and extract the label to add more info here.